Resources/Prototypes/Entities/Constructible/Power/Engines/PA/particles.yml

```
@@ -20,7 +20,8 @@
           !type:PhysShapeAabb
             bounds: "-0.48,-0.48,0.48,0.48"
         hard: false
-        layer: [None]
-        mask:
-        - MobMask
+        layer:
+          # Has to hit mobs, singularity, and singularity generator
+          - MobMask
+        mask: [None]
```

```suggestion
        layer: [None]
        mask:
          # Has to hit mobs, singularity, and singularity generator
          - MobMask
          - Opaque
```
This allows the particle to collide with the Singularity Generator (Opaque) and the Gravitational Singularity (Impassable), as well as mobs.